src/book/03-guides/03-hardware/04-DARwIn-OP2-guide.mdx

```diff
@@ -0,0 +1,184 @@
+
+   5. Clonezilla
+
+      - Select default option **'device-image work with disks or partitions using images'**
+
+   6. Mount Clonezilla Image Directory
+
+      - Select the following **'skip Use existing/home/partimag (Writable device. E.g. hard disk/USB drive)'**
+
+   7. Clonezilla - OpenSource Clone System (OCS)
+
+      - Select the following **'Beginner Beginner mode'** then **'Accept the default options'**
+
+   8. Clonezilla Select mode
+
+      - Select the following **'restoredisk Restore_an_image_to_local_disk'**
+
+   9. Clonezilla - OpenSource Clone System (OCS)
+
+      - Choose the image file to restore **'this will be your recovery image'**
+
+   10. Clonezilla - OpenSource Clone System (OCS)
+
+       - Choose the target disk **'this will be your SSD (make sure to check your SSD capacity to see if the recovery files will fit!)'**
+
+   11. Starting to Restore Image to Device
+
+       - Wait until this is complete.
+
+   12. Completed Restoration
+
+       - Press enter to continue
+
+   13. Power Off or Reboot
+
+       - Select one of the following options **'(0) Power Off, (1) Reboot'**
+
+   14. When the power off have successfully shut the DARwIn down, remove the USB and reboot the robot.
+
+For more information about this process check out [Robotis's Clonezilla recovery page](https://emanual.robotis.com/docs/en/platform/op/maintenance/).
+
+### Calibrating the DARwIn OP2 (Ball Calibration)
+
+1. Connect the Darwin via ethernet onto a laptop on the same internet.
+
+2. Using a portable monitor, plug in a HDMI to mini HDMI cable (mini is to be connect to the Darwin, normal HDMI being connected to the monitor).
+
+3. Start the Darwin so the standard demo program is running. Once booted, and in the initial pose, sitting.
+
+4. Connect a keyboard and mouse to the Darwin and use them to connect to the robot via SSH in the terminal: `SSH robotis@192.168.123.1`, **with password "111111" (six ones, no quotation marks)**
+
+5. Once you are connected to the Darwin, use a browser (either chrome or firefox) to talk to the robot. The web address should be (http://192.168.123.1:8080/)
+
+6. While on this web address you should see the camera's live feed, with camera parameters on the side and bottom of image.
+
+7. The important parameter is the ball (you will find this is the Darwin transport case. It is orange in colour).
+   You will see the section of the camera view highlight pixels in orange, this is showing you an output of its applied color look up table for the ball. You don't want too many of these random
+   pixels around if there is no ball in front of the robot, but once you place a ball in front of the robot, it should see it with the orange applied over the ball.
+
+8. Once you are happy with the calibration, press save and exit the program.
+
+For more information on camera Calibration follow this [Link](https://emanual.robotis.com/docs/en/platform/op2/getting_started/#camera-calibration)
```

Comment:
```suggestion
For more information on [camera calibration follow this link](https://emanual.robotis.com/docs/en/platform/op2/getting_started/#camera-calibration).
```
It's good to make links cover more text, as just having 'link' linked may be difficult for people to click. Having a larger linked component is more accessible.